Doc/library/profile.rst

```diff
@@ -393,7 +393,7 @@ Analysis of the profiler data is done using the :class:`~pstats.Stats` class.
       +------------------+---------------------+----------------------+
       | ``'filename'``   | N/A                 | file name            |
       +------------------+---------------------+----------------------+
-      | ``'module'``     | SortKey.MODULE      | file name            |
+      | ``'module'``     | SortKey.FILENAME    | file name            |
```

Comment:
Move `SortKey.FILENAME` up one line to the `'filename'` entry, change the `'module'` entry to `N/A`, and we're done!  Thank you!